tsconfig.json

```
@@ -13,5 +13,5 @@
         "stripInternal": true
     },
     "include": ["src"],
-    "exclude": ["src/lualib"]
+    "exclude": ["src/lualib", "helpers"]
```

```suggestion
    "exclude": ["src/lualib"]
```
It's out of `include`